cmd/prometheus/main.go

```diff
@@ -1935,12 +1936,12 @@ func (opts agentOptions) ToAgentOptions(outOfOrderTimeWindow int64) agent.Option
 	}
 }
 
-// rwProtoMsgFlagParser is a custom parser for config.RemoteWriteProtoMsg enum.
+// rwProtoMsgFlagParser is a custom parser for remoteapi.MessageTypes enum.
```

Comment:
If we talk about enum we likely have to put a single type name:

```suggestion
// rwProtoMsgFlagParser is a custom parser for remoteapi.WriteMessageType enum.
```

Reply:
BTW (unrelated):  We should probably rename client_golang `remoteapi.MessageTypes` to `remoteapi.WriteMessageTypes` too (we only renamed single type)